src/lib.rs

```diff
@@ -1109,31 +1116,6 @@ impl EGraph {
         }
     }
 
-    // Extract an expression from the current state, returning the cost, the extracted expression and some number
-    // of other variants, if variants is not zero.
-    pub fn extract_expr(&mut self, e: Expr, num_variants: usize) -> Result<ExtractReport, Error> {
```

Comment:
This function was previously used in a number of places before #176 and was now only being used in the un-tested Output command.